src/applications/vaos/utils/selectors.js

```diff
@@ -43,6 +40,17 @@ import {
   getVARFacilityId,
 } from '../services/appointment';
 
+const AUDIOLOGY_TYPES_OF_CARE = [
```

Comment:
All the types of care are in the constants file, I'd leave this in there to be consistent and make them easier to find